strata.default.lua

```diff
@@ -112,6 +104,33 @@ strata.set_config {
 	},
 
 	-- I guess we can have the same systems as above here for convenience for most users:
-	bindings = { ... }, -- same thing as the argument for `strata.set_bindings()`
-	rules = { ... }, -- same thing as the argument for `strata.set_rules()`
+	bindings = {
+		{
+			keys = { "CTRL", "SHIFT", "Q" },
+			cmd = close_all_windows,
+		},
+		{
+			keys = { "WIN", "RETURN" },
+			-- `strata.cmd` should contain a bunch of common built-in functions that generate callbacks to be used
+			-- here, so that the boilerplate is reduced for most use-cases.
+			cmd = strata.cmd.spawn("kitty --title Terminal"),
+		},
+	}, -- same thing as the argument for `strata.set_bindings()`
+	rules = {
+		{
+			triggers = { { event = "win_open_pre", class_name = "firefox" } },
+			action = function(window) window.send_to_workspace(1) end,
+		},
+
+		-- A single action that can be ran from multiple triggers.
+		{
+			triggers = {
+				-- Set mpv to floating, always
+				{ event = "win_open_pre", class_name = "mpv" },
+				-- Set terminals to floating only if on the workspace 1 (where we put firefox previously)
+				{ event = "win_open_pre", workspace = 1, class_name = "kitty" },
+			},
+			action = function(window) window.set_floating() end,
+		},
+	}, -- same thing as the argument for `strata.set_rules()`
```

Comment:
I'm fine with handling this, everything within the `set_config` function (we can just call it multiple times if we want to do things separately), but I'm afraid it will *add* clutter rather than remove some when the helper functions are implemented. Again, no reason to block on this account, but I'll show you what I mean in another draft PR for the lua code.